Error when packaging uber jar

Running `mvn package -Puber -DskipTests` on `master` at [head](https://github.com/hammerlab/guacamole/tree/f052ce3909cfe2a0670a627b4b5d55e98ae22305)  yields,

```
$ mvn package -Puber -DskipTests

...

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-shade-plugin:2.1:shade (deps) on project guacamole: Error creating shaded jar: Error in ASM processing class htsjdk/samtools/BAMIndexer.class: 2560 -> [Help 1]
```

Full stack trace: [gist](https://gist.github.com/e5c/94dd6a5b84a85a2f751b7e314c01ec49)

Maven version info:
```
$ mvn -V

Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T11:41:47-05:00)
Maven home: /hpc/users/change07/apache-maven-3.3.9
Java version: 1.8.0_91, vendor: Oracle Corporation
Java home: /demeter/users/change07/bin/jdk1.8.0_91/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-327.4.5.el7.x86_64", arch: "amd64", family: "unix"
```

However `mvn package -Pguac,deps -DskipTests` works.

--- a/pom.xml
+++ b/pom.xml
@@ -35,7 +35,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>2.1</version>
+        <version>2.4.3</version>
         <configuration>
           <transformers>
             <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">

